src/Template/Element/routes_panel.ctp

```diff
@@ -15,6 +15,9 @@ $routes = Cake\Routing\Router::routes();
     </thead>
     <tbody>
     <?php foreach ($routes as $route): ?>
+        <?php if (strpos($route->getName(), 'debugkit.') === 0) {
+            continue;
```

Comment:
Perhaps we should output them in a muted color instead of hiding entirely?

Reply:
That sounds better

Reply:
then u need to scroll one page. why would anyone be interested in the help tools routes? maybe have a toggle  button to include them and by default exlude? or move to the end.

Reply:
How about a configuration setting that is enabled or disabled by default which hides them as @dereuromark proposes?

Reply:
Scrolling is dependent on the number of routes in the app. I don't think we can make decisions based on whether or not people will need to scroll.

Moving debugkit routes to the end is an option as well.

Reply:
So are we OK with at least graying them out? #999 or sth?

Reply:
Displaying in grey sounds good to me.